### DISCUSSION TITLE
Get specific version of parameter from parameter store

### DISCUSSION THREAD

**Discussion by sector101010** [awspring/spring-cloud-aws#181] [Q&A] [answered]
Is it possible to get a specific version of a parameter? These both return the latest value:
```
@Value("${my.param:1}")
@Value("${my.param:2}")
```

**Comment by MatejNedic** [comment, marked as answer]
Hey @nilfric ,
no, currently it is not possible since parameter store integration is loading parameters by the path. Since we are loading all parameters found on that path. If we were loading parameters by name then we would be able to do so since we can send name:version with AWS SDK. If there is interest from the community and if this is something worth having, we can make an issue and introduce it into Spring Cloud AWS.
